README.md

```diff
@@ -55,6 +55,32 @@ Next page tester will take care of:
 | **useDocument** (experimental) | Render [Document component][next-docs-custom-document]        | `boolean`          | `false`         |
 | **nextRoot**                   | Absolute path to Next.js root folder                          | `string`           | _auto detected_ |
 
+## Test helpers
+
+Next page tester provides an [utility to setup the expected JSDOM environment](/src/testHelpers.ts).
```

Comment:
I would mention why we are providing this. Something about how `NextJS` uses all those features that are not supported in JSDOM and would just put your console.error on fire 🔥

Reply:
✅